compiler/syntax/src/res_ast_debugger.ml

```diff
@@ -584,12 +584,15 @@ module SexpAst = struct
             expression expr;
             Sexp.list (map_empty ~f:case cases);
           ]
-      | Pexp_try (expr, cases) ->
+      | Pexp_try (expr, cases, finally_expr) ->
```

Comment:
I keep on forgetting why we have this debugger file. Gives maintenance burden but can't remember why we need it.